hydrosdk/cluster.py

```diff
@@ -11,30 +11,33 @@ class Cluster:
     Cluster responsible for server interactions
     """
     @staticmethod
-    def connect(address):
+    def connect(http_address, grpc_address=None):
         """
         The preferable factory method for Cluster creation. Use it.
 
-        :param address: connection address
+        :param http_address: connection address
         :raises ConnectionError: if no connection with cluster
         :return: cluster object
 
         Checks the address, the connectivity, and creates an instance of Cluster.
         """
-        cl = Cluster(address)
-        logging.info("Connecting to {} cluster".format(cl.address))
+        cl = Cluster(http_address=http_address, grpc_address=grpc_address)
+        logging.info("Connecting to {} cluster".format(cl.http_address))
         info = cl.build_info()
         if info['manager']['status'] != 'Ok':
-            raise ConnectionError("Couldn't establish connection with cluster {}. {}".format(address, info['manager'].get('reason')))
+            raise ConnectionError("Couldn't establish connection with cluster {}. {}".format(http_address, info['manager'].get('reason')))
         logging.info("Connected to the {} cluster".format(info))
         return cl
 
-    def __init__(self, address):
+    def __init__(self, http_address, grpc_address=None):
         """
         Cluster ctor. Don't use it unless you understand what you are doing.
+        :param http_address:
+        :param grpc_address:
         """
-        parse.urlsplit(address)  # check if address is ok
-        self.address = address
+        parse.urlsplit(http_address)  # check if address is ok
```

Comment:
⚠️  `parse.urlsplit` here makes sure that we don't accept gibberish string as host. 
Do we need the same kind of input validation for `grpc_address` ?

Reply:
Added validation